src/__mocks__/terminalApi/sync.ts

```diff
@@ -365,3 +365,24 @@ export const syncRefund = {
         }
     }
 };
+
+export const syncResEventNotification = {
+    SaleToPOIRequest: {
```

Comment:
We are using this as a mock response, why does it have a "request" structure? The other mocks in this file use another `SaleToPOIResponse` structure.